### ISSUE TITLE
Test codebase with greyscale images

### ISSUE THREAD

**Issue by chebroluharika** [flow2ml/Flow2ML#41]
Test codebase with greyscale images as input, and also fix bugs if there are any.

**Comment by Preeti2095** [comment]
I would like to work on this issue.

**Comment by chebroluharika** [comment]
@Preeti2095 Please goahead and work on this and do please let us know 
when can we expect the deliverable?

**Comment by chebroluharika** [comment]
@Preeti2095 Do we have any update on this?

**Comment by Preeti2095** [comment]
Yes I am doing it, I am getting some errors that's why it is taking me a bit of time. I will complete it soon.

**Comment by Preeti2095** [comment]
You can confirm this with @yvkrishna.

**Comment by Preeti2095** [comment]
I am getting so much of error don't know why, so I am not working in this issue now @chebroluharika. Thank you for letting me work in this issue and sorry for disappointing you mentors.

**Comment by chebroluharika** [comment]
@Preeti2095 you will get errors when you test with 2 channel images. Thats expected. Right now code works for 3 channel data.
